src/framework/mlt.vers

```diff
@@ -480,4 +480,7 @@ MLT_0.9.10 {
   global:
     mlt_factory_repository;
     mlt_properties_to_utf8;
+    mlt_slices_init;
+    mlt_slices_close;
+    mlt_slices_run;
 } MLT_0.9.8;
```

Comment:
I am sorry there was some miscommunication in my previous pull request comment about the changes in this file. You were on the right track, but using the old versioning scheme, and I mis-communicated about MLT_0.9.10. This needs to look like this:
```
MLT_6.4.0 {
  global:
    mlt_slices_init;
    mlt_slices_close;
    mlt_slices_run;
} MLT_0.9.10;
```
The versioning changed recently to 6.x.y, but the symbols in MLT_0.9.10 were added between 0.9.8 and 6.0.0, and I did not want to change the symbol version block on people using git master. But, any new symbols now should go into a new block and use the new versioning scheme.

Feel free to fix this and force push your branch if needed to prevent the need to make a new branch or commit on top of this. Whatever works for you.